element.js

```diff
@@ -6,7 +6,11 @@ module.exports = getFormData
 
 function buildElems(rootElem) {
     var hash = {}
-
+    if (rootElem.name) {
+    	hash[rootElem.name] = rootElem
+    	return hash
```

Comment:
I guess that's opening a whole kettle of fish we don't want to deal with though - I'll remove it :)